components/code-editor/index.js

```diff
@@ -11,6 +11,20 @@ import CodeEditor from './editor';
 import Placeholder from '../../packages/components/src/placeholder';
 import Spinner from '../../packages/components/src/spinner';
 
+/**
+ * Moddule constants
+ */
+let siteURL;
+
+/**
+ * Configure the site's URL to lazy load scripts and styles.
+ *
+ * @param {string} url Site url.
+ */
+export function unstable__setSiteURL( url ) { // eslint-disable-line camelcase
```

Comment:
Just a thought: I wish there was an easy way to package the unstable pieces in one place, _ala_ python's `__future__` module. The more I think about that, the more convinced I am using the `unstable__*` prefix is the right trade-off, though.

Publishing a `@wordpress/unstable` package could lead to some potential conflicts due the dependance on other  `@wordpress` packages to implement the unstable features. Publishing `@wordpress/components-unstable` for every package we maintain is a logistic burden. Package those APIs in subpaths such as `components/unstable`, `data/unstable`, etc would expose them at the root level path anyway.

Reply:
To centralize the unstable APIs in a way that's clearly communicated and maintained. At the moment, we are using different name patterns (`unstable__SomeThing`, `unstableSomething`, and can see how in the future someone could also add `unstable_SomeThing`) which it doesn't scale well.

Reply:
Don't forget `__experimentalSomething`

https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#experimental-apis

😆

Reply:
Has already happened! :sweat_smile: